### PR TITLE
fixed bugs with audio meta data and volume

### PR DESCRIPTION
**Pull request by paynejacob**

### CODE REVIEW

File: points_tracker/views.py [hunk 1]
```diff
@@ -111,24 +112,24 @@ def upload_file():
...
+                    audiofile = AudioSegment.from_mp3(filepath)
                 elif extension == ".wav":
-                    length = 0
+                    audiofile = AudioSegment.from_wav(filepath)
                 else:
-                    length = 0
+                    audiofile = AudioSegment.from_file(filepath)
+
+                audiofile = audiofile + -(audiofile.dBFS - current_app.config['AUDIO_DB_LEVEL'])    #level the file
+                audiofile = audiofile[:current_app.config['MAX_AUDIO_DURATION']]                    #trim the file
```
Comment (paynejacob) [inline points_tracker/views.py:124]: this should fix our issue with people uploading long files it just keeps the first n seconds of the file if its too long

File: manage.py [hunk 2]
```diff
@@ -55,6 +55,32 @@ def create_user():
     User.create(username=user, password=p1, active=True, is_admin=True)
     print 'Administrator account created for {}'.format(user)
 
+@manager.command
+def normalize_audio_files():
```
Comment (paynejacob) [inline manage.py:59]: this will convert our current files so we don't have to re add them

File: points_tracker/views.py (hunk 1 above)
Comment (bll-z) [inline points_tracker/views.py:124]: z

File: manage.py [hunk 3]
```diff
@@ -55,6 +55,32 @@ def create_user():
...
+    import wave
+    import contextlib
+    from pydub import AudioSegment
+
+    #get all files
+    print 'Querying file list...'
+    audiofiles = Audio.query.all()
+
+    print 'Processing '+str(len(audiofiles))+' files...'
```
Comment (wmarshall) [inline manage.py:68]: Minor gripe: use str.format to make less ugly

File: points_tracker/settings.py [hunk 4]
```diff
@@ -65,6 +65,9 @@ class DevConfig(Config):
     PASSWORD_REQUIRE_NUMBER = False
     UPLOAD_FOLDER = os.path.abspath(os.path.join(Config.PROJECT_ROOT, 'points_tracker/static/audio'))
 
+    AUDIO_DB_LEVEL = -21.9297220851 #after this setting is changed you should run normalize_audio_files from manage.py
+    MAX_AUDIO_DURATION = 15*1000 #this is in ms
```
Comment (wmarshall) [inline points_tracker/settings.py:69]: Append _MS to constant to denote units?

File: points_tracker/views.py [hunk 5]
```diff
@@ -111,24 +112,24 @@ def upload_file():
...
-                    length = int(round(MP4(filepath).info.length))
-                    sound = AudioSegment.from_file(filepath)
-                    sound.export(filepath, format="wav")
+                    audiofile = AudioSegment.from_mp3(filepath)
                 elif extension == ".wav":
-                    length = 0
+                    audiofile = AudioSegment.from_wav(filepath)
                 else:
-                    length = 0
+                    audiofile = AudioSegment.from_file(filepath)
```
Comment (wmarshall) [inline points_tracker/views.py:121]: Why not replace this if with 

```
audiofile = AudioSegment.from_file(filepath, filepath.split(".")[-1])
```

?

**Pull request merged**